openeo/rest/connection.py

```diff
@@ -719,6 +719,24 @@ def list_processes(self, namespace: str = None) -> List[dict]:
             processes = self.get('/processes/' + namespace, expected_status=200).json()["processes"]
         return VisualList("processes", data=processes, parameters={'show-graph': True, 'provide-download': False})
 
+    def describe_process(self, id: str, namespace: str = None) -> dict:
+        """
+        Retuens a single process from the back end.
+
+        :param id: The id of the process.
+        :param namespace: The namespace of the process.
+
+        :return: process: Dict The process definition.
```

Comment:
I know you copied this from `list_processes` (were it is also wrong), but this `Dict` does not belong there, the doc generator tool gets that from the function return type hint. It should just be

> :return: The process definition.

Reply:
```suggestion
        :return: The process definition.
```